BraveWallet/Crypto/Portfolio/AddCustomAssetView.swift

```diff
@@ -32,6 +32,17 @@ struct AddCustomAssetView: View {
           header: WalletListHeaderView(title: Text(Strings.Wallet.tokenContractAddress))
         ) {
           TextField(Strings.Wallet.enterContractAddress, text: $addressInput)
+            .onChange(of: addressInput) { newValue in
+              if !newValue.isEmpty {
+                userAssetStore.tokenInfo(by: newValue) { token in
```

Comment:
We probably want some sort of loading indicator if we hit a server for info (I think one branch in tokenInfo hits the server)

Reply:
We also may want to cancel it or not fill if the user enters info into the other fields before it finishes

Reply:
Desktop will override the token info if they are filled-in before this. But i can change it and update the test plan.

Reply:
I guess I just want to avoid you pasting an address in and then you go start typing into another field and _boom_ it replaces it when the network call finishes while you're typing and then you're still typing and everything's all messed up lol

Reply:
yeah i add a fix to disable the textfield while we are looking for a matching token with a loader at the bottom.